Public/Scripts/Common.js

```diff
@@ -0,0 +1,47 @@
+export const preventFormSubmit = (id, submitFunction) => {
+    window.addEventListener("DOMContentLoaded", () => {
+        const form = document.getElementById(id)
+        if (form == null) { return }
+
```

Comment:
```suggestion

```
This should throw an error now, it shouldn't be invoked if the form is not present.